SimDataFormats/CaloTest/src/classes.h

```diff
@@ -1,8 +1,15 @@
 #include "SimDataFormats/CaloTest/interface/HcalTestHistoClass.h"
+#include "SimDataFormats/CaloTest/interface/ParticleFlux.h"
 #include "DataFormats/Common/interface/Wrapper.h"
 
 namespace SimDataFormats_CaloTest {
   struct dictionary {
     HcalTestHistoClass theHcalTestHistoClass;
+    ParticleFlux                                         d1;
```

Comment:
@bsunanda - please, use consistent naming convention for private variables: e.g. trailing underscore or leading "m_"